webapp/channels/src/components/admin_console/permission_schemes_settings/permission_team_scheme_settings/team_in_list/team_in_list.tsx

```diff
@@ -16,41 +16,39 @@ type Props = {
     isDisabled: boolean;
 }
 
-export default class TeamInList extends React.PureComponent<Props> {
-    handleRemoveTeam = () => {
-        const {team, isDisabled, onRemoveTeam} = this.props;
+const TeamInList = ({team, isDisabled, onRemoveTeam}: Props) => {
+    const handleRemoveTeam = useCallback(() => {
         if (isDisabled) {
             return;
         }
         onRemoveTeam(team.id);
-    };
-
-    render() {
-        const {team, isDisabled} = this.props;
-        return (
-            <div
-                className='team'
-                key={team.id}
-            >
-                <div className='team-info-block'>
-                    <TeamIcon
-                        content={team.display_name}
-                        url={imageURLForTeam(team)}
-                    />
-                    <div className='team-data'>
-                        <div className='title'>{team.display_name}</div>
-                    </div>
+    }, [isDisabled, team.id, onRemoveTeam]);
+
+    return (
+        <div
+            className='team'
+            key={team.id}
+        >
+            <div className='team-info-block'>
+                <TeamIcon
+                    content={team.display_name}
+                    url={imageURLForTeam(team)}
+                />
+                <div className='team-data'>
+                    <div className='title'>{team.display_name}</div>
                 </div>
-                <a
-                    className={isDisabled ? 'remove disabled' : 'remove'}
-                    onClick={this.handleRemoveTeam}
-                >
-                    <FormattedMessage
-                        id='admin.permissions.teamScheme.removeTeam'
-                        defaultMessage='Remove'
-                    />
-                </a>
             </div>
-        );
-    }
-}
+            <a
+                className={isDisabled ? 'remove disabled' : 'remove'}
```

Comment:
Can we change this to use `classnames`? (you can see how it is used in other files in the codebase)

If I remember correctly, it should be something like `classnames('remove', {disabled: isDisabled})`.